web/packages/teleport/src/SamlApplications/useSamlAppActions.tsx

```diff
@@ -88,7 +88,15 @@ export interface SamlAppAction {
 export const SamlAppActionContext = createContext<SamlAppAction>(null);
 
 export function useSamlAppAction() {
-  return useContext(SamlAppActionContext);
+  const context = useContext(SamlAppActionContext);
+
+  if (!context) {
+    throw new Error(
+      'useSamlAppAction must be used within a SamlAppActionProvider'
+    );
+  }
+
+  return context;
```

Comment:
This change should not change the behavior of the hook. Previously if there was no context, the callsite would throw an error while trying to destructure the return value. Now the function itself is going to throw a more meaningful error.